scripts/setup_gae.sh

```diff
@@ -46,3 +46,5 @@ if [ ! -d "$GOOGLE_APP_ENGINE_HOME" ]; then
   unzip -q gae-download.zip -d $TOOLS_DIR/google_appengine_1.9.73/
   rm gae-download.zip
 fi
+
+export SETUP_GAE_DONE=true
```

Comment:
Where is this used?

Reply:
`$SETUP_GAE_DONE` is referenced at line 20: 
https://github.com/oppia/foundation-website/blob/865dff7834ef52ad43877d0f201bf4fe32cba8af/scripts/setup_gae.sh#L20

Reply:
Ah ok, thanks!